docs/src/further_topics/ugrid/operations.rst

```diff
@@ -0,0 +1,985 @@
+        # 'Apply' the region to the PolyData object.
+        >>> region_polydata = region.enclosed(global_polydata, preference="center")
+        # Get the remaining face indices, to use for indexing the Cube.
+        >>> indices = region_polydata["vtkOriginalCellIds"]
+
+        >>> print(type(indices))
+        <class 'numpy.ndarray'>
+        # 101 is smaller than the original 864.
+        >>> print(len(indices))
+        101
+        >>> print(indices[:10])
+        [ 6  7  8  9 10 11 18 19 20 21]
+
+        # Use the face indices to subset the global cube.
+        >>> region_cube = global_cube[:, indices]
+
+        # In this case we **know** the indices correspond to a contiguous
+        #  region, so we will convert the sub-setted Cube back into a
+        #  Cube-with-Mesh.
+        >>> new_mesh = Mesh.from_coords(*region_cube.coords(dimensions=1))
+        >>> new_mesh_coords = new_mesh.to_MeshCoords(global_cube.location)
+        >>> for coord in new_mesh_coords:
+        ...     region_cube.remove_coord(coord.name())
+        ...     region_cube.add_aux_coord(coord, 1)
+
+        # A Mesh-Cube with a subset (101) of the original 864 faces.
+        >>> print(region_cube)
+        surface_convective_rainfall_rate / (kg m-2 s-1) (-- : 72; -- : 101)
+            Mesh coordinates:
+                latitude                                    -        x
+                longitude                                   -        x
+            Auxiliary coordinates:
+                time                                        x        -
+            Cell methods:
+                point                                   time
+            Attributes:
+                Conventions                             UGRID
+                description                             Created by xios
+                interval_operation                      300 s
+                interval_write                          300 s
+                name                                    lfric_ngvat_2D_72t_face_half_levels_main_conv_rain
+                online_operation                        instant
+                timeStamp                               2020-Oct-18 21:18:35 GMT
+                title                                   Created by xios
+                uuid                                    b3dc0fb4-9828-4663-a5ac-2a5763280159
+
+Regridding
+----------
+.. |tagline: regridding| replace:: |different| - use iris-esmf-regrid for mesh regridders
+
+.. rubric:: |tagline: regridding|
+
+Regridding to or from a mesh requires different logic than Iris' existing
+regridders, which are designed for structured grids. For this we recommend
+ESMF's powerful regridding tools, which integrate with Iris' mesh data model
+via the :ref:`ugrid iris-esmf-regrid` package.
+
+.. todo: inter-sphinx links when available.
+
+Regridding is achieved via the
+:class:`esmf_regrid.experimental.unstructured_scheme.MeshToGridESMFRegridder`
```

Comment:
Some of this is sort-of boilerplate, as all Iris regrid schemes basically work the same way.
I also think we need to emphasize that the regridders **_are_** regular Iris "schemes", so the operation of them should be familiar if you've used Iris regridding before
 -- i.e. you don't actually need to know anything much more, except that iris-esmf-regrid holds the functionality you need .  And maybe not to mess about with the olders ones, but I think you already made that clear enough.

Reply:
@stephenworsley help me

Reply:
OK thanks. As it stands I'm not planning to make any changes (?)